### DISCUSSION TITLE
As an Environment Manager, why doesn't tea set me up?

### DISCUSSION THREAD

**Discussion by trentlarson** [pkgxdev/pkgx#265] [Help] [answered]
I've got tea installed, with this at the end of my .zprofile `add-zsh-hook -Uz chpwd(){ source <(tea -Eds) }`

So I add the following to my README.md in a new directory:

```
## Dependencies

|   Project   | Version  |
|-------------|----------|
| nodejs.org  | ^16.18.0 |
| tea.xyz     | ^0       |
```

I expected that I would have that version of node available when I change into that dire, but I don't see a tea prompt and it is just using my default node version. Am I expecting too much?

**Comment by mxcl** [comment, marked as answer]
Sorry I missed this.

You won’t (yet) see a tea prompt, when changing dir, but we intend to add this visual feedback.

What does `tea -Eds` say when you are in that directory?

**Reply by trentlarson** [reply]
Nothing. I get a successful shell result of 0, but no output.

**Reply by mxcl** [reply]
k, could be because there is no `.git` directory? currently we require that tho I totally agree this is (probably) dumb.

**Reply by trentlarson** [reply]
a HA! That totally fixed it: I created an empty .git directory, cd'd out and then back in, and now it's using the node that's stored in my ~/.tea directory. I even changed the README and I see it using whatever I specify in there. 👏
